### ISSUE TITLE
Using the arrow keys to navigate between tabs doesn't work reliably. Sometimes you can't tab to or arrow back to the previous tabs.

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#482]
# Bug Report

## Description
For tabs, using the arrow keys to navigate between tabs doesn't work reliably. Sometimes you can't tab to or arrow back to the previous tabs.

## Steps to Reproduce

1. TBD

## Additional Context / Screenshots
Identified as part of functional verification on [this PR](https://github.com/cerner/terra-core/pull/2004#issuecomment-438396868).

## Expected Behavior
Using the arrow keys to navigate between tabs works reliably.

## Environment

* Component Name and Version: terra-tabs
* Browser Name and Version: IE 11 + JAWS

**Comment by ShettyAkarsh** [comment]
@neilpfeiffer , @bjankord , @JakeLaCombe there is one more issue #483 where using right and left arrow keyboard keys along with JAWS does not read the tab text. but this current issue has been logged for navigation between tabs using the  right and left arrow keys along with JAWS. so is there any standard requirement or expectations for the usage of these keys on tabs across all browsers whether to read the tab text or navigate across ?.   or do we need to have different implementation for navigations across tabs for all the browsers like use tab keyboard keys to navigate and read the tab text when the right and left arrow keys are used ?

Please share your thoughts on this.

**Comment by bjankord** [comment]
I do think we'll need to revisit using the left and right arrow keys to navigate through tabs as JAWS has special treatment for these keys and I'm not sure we should try to override that behavior.

**Comment by bjankord** [comment]
We're chalking this up as a known issue with the current construction of this componet. There is not a passive way to correct this behavior and so we want to enhance the component with a [roving index](https://www.stefanjudis.com/today-i-learned/roving-tabindex/) to better manage focus functionality within the tabs component in conjunction with another enhancement to make the tabs operate as a controlled component: https://github.com/cerner/terra-framework/issues/481

We'll be tracking the accessibility work under https://github.com/cerner/terra-framework/issues/481
